packages/telegram-cloud-storage-stamper/src/index.ts

```diff
@@ -23,7 +23,7 @@ export type CloudStorageAPIKey = {
 };
 
 // Constant for default key name
-export const DEFAULT_TURNKEY_CLOUD_STORAGE_KEY = "TURNKEY_API_KEY";
+const DEFAULT_TURNKEY_CLOUD_STORAGE_KEY = "TURNKEY_API_KEY";
```

Comment:
So this export is used in the demo-telegram-mini app, for deleting the key upon logout [here](https://github.com/tkhq/demo-telegram-mini-app/blob/main/src/app/wallet/page.tsx#L180). However I think we can probably still get away with this if we create some clear/deleteAPIKey() method that accepts a key whose default value is DEFAULT_TURNKEY_CLOUD_STORAGE_KEY